dgbpy/dgbcombine.py

```diff
@@ -0,0 +1,131 @@
+import numpy as np
+import pickle
+import joblib
+import onnxruntime as rt
+import onnx
+from skl2onnx import convert_sklearn
+from skl2onnx.common.data_types import FloatTensorType
+import sclblonnx as so
+
+
+savetypes = ['onnx', 'joblib', 'pickle', 'xgboost']
+modeltypes = ['classification', 'regression']
+defsavetype = savetypes[2]
+defmodeltype = modeltypes[0]
+
+def fit(models, inputs, outputs): # models, Xs, ys
```

Comment:
Maybe nice to have, but does not fit as such in our workflow. You may copy it for reference to dgbscikit.py